chebyshev_distance.go

```diff
@@ -0,0 +1,34 @@
+//
+// Compiled with Go version go1.9 linux/amd64
+// Contributor : Shivendra Mishra 
+// Contact : shivendra_mishra@live.com
+//
+package stats
+
+import (
+	"math"
+)
+
+// Compute Chebyshev distance between two data points 
+func ComputeChebyshevDistance( data_point_x, data_point_y []float64 ) ( distance float64, 
+                                                                        err error ) {
+	var (
+		temp_distance float64
```

Comment:
As mentioned in https://golang.org/doc/effective_go.html#mixed-caps it's preferred to use camelCase for variables and function names (or any names really)